frontend/i18n/en/projects/comments.js

```diff
@@ -2,5 +2,11 @@ export default {
   comments: 'Comments',
   removeComment: 'Remove Comment',
   removePrompt: 'Are you sure you want to remove these comments?',
-  commentView: 'View'
+  commentView: 'View',
+  created_at: 'Created at',
+  document: 'Document',
+  username: 'Username',
+  text: 'Text',
+  send: 'Send',
+  message: 'Message'
```

Comment:
some translations for things like username or document already exist elsewhere. please do not duplicate them here.